src/Microsoft.Health.Fhir.Core/Resources.Designer.cs

```diff
@@ -745,6 +745,24 @@ internal class Resources {
             }
         }
         
+        /// <summary>
+        ///   Looks up a localized string similar to One or more resources to be reindexed were not found, indicating that they have been deleted since the reindex job kicked off..
+        /// </summary>
+        internal static string ReindexingResourceNotFound {
+            get {
+                return ResourceManager.GetString("ReindexingResourceNotFound", resourceCulture);
+            }
+        }
+        
+        /// <summary>
+        ///   Looks up a localized string similar to The version of one or more resources did not match, indicating that they have been updated since the reindex job kicked off..
+        /// </summary>
+        internal static string ReindexingResourceVersionConflict {
+            get {
+                return ResourceManager.GetString("ReindexingResourceVersionConflict", resourceCulture);
+            }
+        }
```

Comment:
These will likely be updated once we catch these errors and attempt to reindex again (please see [AB#79925](https://microsofthealth.visualstudio.com/Health/_workitems/edit/79925)).